### PR TITLE
Fixing PYTHONPATH issue in wrappers (#200)

### PR DESCRIPTION
**Pull request by holtgrewe**

### CODE REVIEW

**Review by eudesbarbosa — APPROVED** [review]

File: snappy_wrappers/wrappers/gcnv/merge_cohort_vcfs/wrapper.py [hunk 1]
```diff
@@ -1,10 +1,16 @@
...
-from snappy_wrappers.tools import vcf_merge_exome_cnvs
+
 import os
 
 from snakemake.shell import shell
 
+# The following is required for being able to import snappy_wrappers modules
+# inside wrappers.  These run in an "inner" snakemake process which uses its
+# own conda environment which cannot see the snappy_pipeline installation.
+base_dir = os.path.normpath(os.path.join(os.path.dirname(__file__), "..", "..", ".."))
```
Comment (ericblanc20) [inline snappy_wrappers/wrappers/gcnv/merge_cohort_vcfs/wrapper.py:10]: Possibly lacks a level going upstream:

```
base_dir = os.path.normpath(os.path.join(os.path.dirname(__file__), "..", "..", "..", ".."))
```

File: snappy_wrappers/wrappers/jannovar_par/annotate_vcf/parallel_annotate_vcf.py [hunk 2]
```diff
@@ -1,16 +1,22 @@
...
 """
 
-from snappy_wrappers.wrapper_parallel import (
+import os
+import sys
+
+# The following is required for being able to import snappy_wrappers modules
+# inside wrappers.  These run in an "inner" snakemake process which uses its
+# own conda environment which cannot see the snappy_pipeline installation.
+base_dir = os.path.normpath(os.path.join(os.path.dirname(__file__), "..", "..", ".."))
```
Comment (ericblanc20) [inline snappy_wrappers/wrappers/jannovar_par/annotate_vcf/parallel_annotate_vcf.py:11]: Possibly lacks a level going upstream:

```
base_dir = os.path.normpath(os.path.join(os.path.dirname(__file__), "..", "..", "..", ".."))
```

File: snappy_wrappers/wrappers/mutect2_par/run/parallel_mutect2.py [hunk 3]
```diff
@@ -1,12 +1,16 @@
...
 """
 
 import os
+import sys
 import textwrap
 
+# The following is required for being able to import snappy_wrappers modules
+# inside wrappers.  These run in an "inner" snakemake process which uses its
+# own conda environment which cannot see the snappy_pipeline installation.
+base_dir = os.path.normpath(os.path.join(os.path.dirname(__file__), "..", "..", ".."))
```
Comment (ericblanc20) [inline snappy_wrappers/wrappers/mutect2_par/run/parallel_mutect2.py:12]: Wroks with an additional level going upstream:

```
base_dir = os.path.normpath(os.path.join(os.path.dirname(__file__), "..", "..", "..", ".."))
```

File: snappy_wrappers/wrappers/varscan_par/call_joint/parallel_call_joint.py [hunk 4]
```diff
@@ -1,12 +1,16 @@
...
-isort:skip_file
 """
 
 import os
+import sys
 
+# The following is required for being able to import snappy_wrappers modules
+# inside wrappers.  These run in an "inner" snakemake process which uses its
+# own conda environment which cannot see the snappy_pipeline installation.
+base_dir = os.path.normpath(os.path.join(os.path.dirname(__file__), "..", "..", ".."))
```
Comment (ericblanc20) [inline snappy_wrappers/wrappers/varscan_par/call_joint/parallel_call_joint.py:12]: Possibly lacks a level going upstream:

```
base_dir = os.path.normpath(os.path.join(os.path.dirname(__file__), "..", "..", "..", ".."))
```

**Review by ericblanc20 — CHANGES_REQUESTED** [review]
More general question: is it a potential problem that the python & snakemake versions run the by the parallel wrappers are different from the snappy environment seen by the user?

**Comment by holtgrewe** [comment]
@ericblanc20 thanks, good catches, I just pushed a fixed version

**Review by ericblanc20 — APPROVED** [review]

**Pull request merged**